packages/taquito-ledger-signer/src/taquito-ledger-signer.ts

```diff
@@ -178,7 +178,10 @@ export class LedgerSigner implements Signer {
     messageToSend = chunkOperation(messageToSend, watermarkedBytes2buff);
     const ledgerResponse = await this.signWithLedger(messageToSend);
     let signature;
-    if (this.derivationType === DerivationType.ED25519 || this.derivationType === DerivationType.BIP32_ED25519) {
+    if (
+      this.derivationType === DerivationType.ED25519 ||
+      this.derivationType === DerivationType.BIP32_ED25519
+    ) {
```

Comment:
did i miss places that needed bip32 additions?

Reply:
I think this is just a spacing/format changes from lint